Realm/Realm/Realm.cs

```diff
@@ -1334,9 +1332,9 @@ public void WriteCopy(RealmConfigurationBase config)
         {
             Argument.NotNull(config, nameof(config));
 
-            if (Config is SyncConfiguration originalConfig && config is SyncConfiguration copiedConfig && originalConfig.Partition != copiedConfig.Partition)
+            if (Config is SyncConfiguration originalConfig && config is SyncConfiguration copiedConfig && originalConfig._partition != copiedConfig._partition)
```

Comment:
I understand why it was done, it just looks a little funny with the underscore in front of the property.

Reply:
It's because that's a field, not the actual property. Once we release a new major version, we can replace the old property with the field and that will look a lot nicer.